lib/neo4j/core/cypher_session.rb

```diff
@@ -1,7 +1,10 @@
+require 'active_support'
```

Comment:
Generally we've been `require`ing just the parts of `active_support` that are needed.  [For example](https://github.com/neo4jrb/neo4j-core/blob/master/lib/neo4j/core/node.rb#L2)